worldedit-core/src/main/java/com/sk89q/worldedit/command/GeneralCommands.java

```diff
@@ -76,6 +85,62 @@
 @CommandContainer(superTypes = CommandPermissionsConditionGenerator.Registration.class)
 public class GeneralCommands {
 
+    public static void register(CommandRegistrationHandler registration,
+                                CommandManager commandManager,
+                                CommandManagerService commandManagerService,
+                                WorldEdit worldEdit) {
+        // Collect the tool commands
+        CommandManager collect = commandManagerService.newCommandManager();
+
+        registration.register(
+            collect,
+            GeneralCommandsRegistration.builder(),
+            new GeneralCommands(worldEdit)
+        );
+
+
+        Set<org.enginehub.piston.Command> commands = collect.getAllCommands()
+            .collect(Collectors.toSet());
+        for (org.enginehub.piston.Command command : commands) {
+            if (command.getName().equals("/fast")) {
+                // deprecate to `//perf`
+                commandManager.register(CommandUtil.deprecate(
+                    command, "//fast duplicates //perf " +
```

Comment:
//fast serves a different purpose for FAWE. The code should be adjusted to line that out.

Reply:
I commented this part out, therefore the method below isn't called. I guess we can keep this code around until it's removed from WE anyways.